fxmanifest.lua

```diff
@@ -1,17 +1,23 @@
 fx_version 'cerulean'
 game 'gta5'
 
-description 'QB-VehicleFailure'
+description 'https://github.com/Qbox-project/qbx-vehiclefailure'
 version '1.0.0'
 
 shared_scripts {
+    '@qbx-core/import.lua',
     '@ox_lib/init.lua',
-    '@qb-core/shared/locale.lua',
+    '@qbx-core/shared/locale.lua',
     'locales/en.lua',
     'config.lua'
 }
 
 client_script 'client.lua'
 server_script 'server.lua'
 
-lua54 'yes'
+modules {
+	'qbx-core:core',
+    'qbx-core:utils'
+}
+
+lua54 'yes'
```

Comment:
Add `use_experimental_fxv2_oal 'yes'` here to use oal, it lazy loads natives and in turn speeds up the resource (or well, lower its usage)